packages/google_maps_flutter/google_maps_flutter/example/integration_test/google_maps_test.dart

```diff
@@ -15,6 +16,12 @@ import 'src/tiles_inspector.dart' as tiles_inspector;
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
+  // TODO(bparrishMines): These tests are failing on web due to an error being
+  // thrown after completion. See https://github.com/flutter/flutter/issues/145149
+  if (isWeb) {
```

Comment:
I know it'll be a lot more code, but I think it would be better to do something like:

```dart
// TODO(bparrishMines): Tests are failing on web due to an error being
// thrown after completion. See https://github.com/flutter/flutter/issues/145149
const bool _skipOnWeb = isWeb;

[ Add skip: _skipOnWeb to every test ]
```

That way they it'll be obvious both when looking at the test, and from the test run output, that web is skipping tests, vs them being silently skipped in an unusual place.